tests/resources/deeprec/output.txt

```diff
@@ -0,0 +1,7 @@
+2.316978e-06
```

Comment:
All the resource files (npy, txt and other data), could you please upload it a blob? We don't want the repo to be super heavy. The blob we use is called recodatasets, I just created a folder called deeprec. I'll send you the keys privately

Reply:
Can you provide some examples for blob usage? I am not very clear what is it

Reply:
A good example is on this notebook: https://github.com/Microsoft/Recommenders/blob/staging/notebooks/02_model/ncf_deep_dive.ipynb

The diagram with the architecture of NCF is actually hosted here: https://recodatasets.blob.core.windows.net/images/NCF.svg. 

Another example are the movielens datasets, we load them from an external storage, you can see the code here: https://github.com/Microsoft/Recommenders/blob/staging/reco_utils/dataset/movielens.py#L386

Reply:
Got it, thanks!

Reply:
I separate data files to blob storage,  you can see the commit here: https://github.com/Microsoft/Recommenders/pull/430/commits/4eb799c52114985ace545f3849ea396477f8db9e